src/component/misc/EditableComponent.ts

```diff
@@ -22,4 +23,16 @@ export default class EditableComponent<P = {}, S extends EditableComponentState
     public onCloseEdit = () => {
         this.setState({edit: false});
     };
+
+    protected onRemoveClick = () => {
+        this.setState({showRemoveDialog: true});
+    };
+
+    public onCloseRemove = () => {
+        this.setState({showRemoveDialog: false});
+    };
+
+    protected onRemoveCancel = () => {
```

Comment:
Is it necessary to duplicate `onCloseRemove` with a different name? Why not just reuse it for the cancel callback?